src/server/api/routers/experiments.router.ts

```diff
@@ -4,11 +4,37 @@ import { prisma } from "~/server/db";
 
 export const experimentsRouter = createTRPCRouter({
   list: publicProcedure.query(async () => {
-    return await prisma.experiment.findMany({
+    const experiments = await prisma.experiment.findMany({
       orderBy: {
         sortIndex: "asc",
       },
     });
+
+    const experimentsWithCounts = await Promise.all(
```

Comment:
This is ugly (vs doing an aggregate query) but we're only running this from the experiments index page now, not every page? Should be fine in that case.

Reply:
I believe that is the case. I'll add a TODO.